Alembic/config.py

```diff
@@ -22,8 +22,12 @@
 			" -D HDF5_ROOT={buildDir}"
 			" -D ALEMBIC_PYILMBASE_INCLUDE_DIRECTORY={buildDir}/include/OpenEXR"
 			" -D USE_HDF5=TRUE"
-			" -D USE_PYILMBASE=TRUE"
-			" -D USE_PYALEMBIC=TRUE"
+			# Temporarily disabling python bindings because the Alembic
+			# setup isn't savvy to Boost's renaming of `libboost_python.so`
+			# to `libboost_python37.so`. Let's see how much other projects are
+			# screwed by this before deciding on a solution.
+			#" -D USE_PYILMBASE=TRUE"
+			#" -D USE_PYALEMBIC=TRUE"
```

Comment:
Do we need to leave dead code here and below (we're not doing it for USD).

Reply:
6153f7e